static/js/dashboard.js

```diff
@@ -1130,9 +1130,22 @@ function addDbSettingsEventListeners() {
 }
 
 function saveDbSetting() {
+    let trimmedDbName = $('.dbSet-dbName').val().trim();
+    
+    if (!trimmedDbName) {
+        // Show error message or handle the case where the name is empty
+        alert('Dashboard name cannot be empty or consist of only spaces.');
```

Comment:
change the message to  `Dashboard name cannot be empty`

Reply:
Ok so we don't have to mention about white-spaces in message.